x-pack/test/api_integration/apis/metrics_ui/infra.ts

```diff
@@ -269,5 +270,38 @@ export default function ({ getService }: FtrProviderContext) {
         );
       });
     });
+
+    describe('Host with active alerts', () => {
+      before(async () => {
+        await Promise.all([
+          esArchiver.load('x-pack/test/functional/es_archives/infra/alerts'),
+          esArchiver.load('x-pack/test/functional/es_archives/infra/metrics_and_logs'),
+        ]);
+      });
+
+      after(async () => {
+        await Promise.all([
+          esArchiver.unload('x-pack/test/functional/es_archives/infra/alerts'),
+          esArchiver.unload('x-pack/test/functional/es_archives/infra/metrics_and_logs'),
+        ]);
+      });
+
+      describe('fetch hosts', () => {
+        it('should return metrics for a host with alert count', async () => {
+          const body: GetInfraMetricsRequestBodyPayload = {
+            ...basePayload,
+            range: {
+              from: '2018-10-17T19:42:21.208Z',
```

Comment:
Nit: You can move the dates to the constants and use them similarly to the functional tests [constants](https://github.com/elastic/kibana/blob/03f529b478b2542630dcf2a434f86a2365245904/x-pack/test_serverless/functional/test_suites/observability/infra/constants.ts#L18-L19) for the dates and use name them so we know that we use those dates for the alerts data